hs_core/management/commands/update_all_resource_files_system_metadata.py

```diff
@@ -0,0 +1,53 @@
+from django.conf import settings
```

Comment:
It seems it will take about 3 days to finish running this command. Will delete this command.